runtime/lib/ttnn/program.cpp

```diff
@@ -174,6 +177,22 @@ run(::tt::target::ttnn::ToMemoryConfigOp const *op, ::ttnn::Device &device,
   }
 }
 
+static void
+run(::tt::target::ttnn::EmptyOp const *op, ::ttnn::device::Device &device,
+    std::unordered_map<std::uint32_t, ::ttnn::Tensor *> &liveTensors,
+    std::list<::ttnn::Tensor> &tensorPool) {
+
+  ::ttnn::DataType targetDataTypeTTNN = utils::toTTNNDataType(
+      op->out()->desc()->layout()->memory_desc()->data_type());
+  // TODO: determine layout, hardcoding tile_layout for now
+  auto desiredLayout = ::ttnn::Layout::TILE;
+  auto shape = ::ttnn::Shape(::tt::tt_metal::Shape(
+      utils::toShapeFromFBShape(*op->out()->desc()->shape())));
+  tensorPool.push_back(
+      ::ttnn::empty(shape, targetDataTypeTTNN, desiredLayout, device));
+  liveTensors.try_emplace(op->out()->global_id(), &tensorPool.back());
+}
+
```

Comment:
@jnie-TT @tapspatel

Guys, would you mind checking this piece of code please?

I'd also like to know if you have any suggestions on handling the layout (tile vs row-major) here - I hardcoded it as I didn't find it in the structs of the output `TensorRef`.

Reply:
@svuckovicTT this is related to #272, the idea is once we emit the correct tile dims, we can determine the layout (1x1 untilized, 32x32 tilized)

Reply:
Gotcha, so I'll leave it as-is for the time being.